lib/config/validation.ts

```diff
@@ -697,3 +706,61 @@ function validateRegexManagerFields(
     }
   }
 }
+
+// basic validation for global config options
```

Comment:
Can this refactoring be done separately/first?

Reply:
Okay 👍

Reply:
https://github.com/renovatebot/renovate/pull/25218